garden_ai/pipelines.py

```diff
@@ -43,6 +43,51 @@
 logger = logging.getLogger()
 
 
+class Repository(BaseModel):
+    """
+    The `Repository` class represents all the metadata we want to \
+    publically expose about the repositories used to build the pipeline.
+
+    Attributes:
+        repo_name (str):
+            A title that the repository can be referenced by.
+        url (str):
+            A link where this repository can be publically viewed.
+        contributors List[str]:
+            Acknowledge contributors to the development of \
+            this repository.
+
+    """
+
+    repo_name: str = Field(...)
+    url: str = Field(...)
+    contributors: List[str] = Field(default_factory=list)
+
+
+class Papers(BaseModel):
```

Comment:
It looks like this class represents just one paper, so it should probably be a singular `Paper` as opposed to `Papers`.